### DISCUSSION TITLE
How to use Bandcamp Plugin?

### DISCUSSION THREAD

**Discussion by sly-fox-git** [beetbox/beets#5116] [Q&A] [answered]
I installed the Bandcamp plugin for Beets, but I can't figure out how to use it to actually tag my media.
I can enter the URL of the album, and it shows me all kinds of data, but what is the actual command to tag an album with Bandcamp data? If I do

`beet import -L Album Name`

The search goes to the MusicBrainz database instead of searching Bandcamp.

**Comment by arsaboo** [comment]
`-L` is to reimport. If you are importing files for the first time, use `beet import <path>`. Make sure you have configured the plugin.

**Reply by sly-fox-git** [reply]
Right but if I re-import or import, beets Bandcamp plugin will automatically tag albums or do I need a special command to tell beets to sue Bandcamp?

**Reply by arsaboo** [reply, marked as answer]
If Bandcamp is configured, it will be used as a tag source. You can use the `-t` (timid) flag so the importer will ask for your input before tagging. You should see Bandcamp as one of the tag sources among all the options.

**Reply by sly-fox-git** [reply]
Thank you. it works now.
